common/src/main/java/com/wynntils/features/user/LobbyUptimeFeature.java

```diff
@@ -7,11 +7,43 @@
 import com.wynntils.core.components.Model;
 import com.wynntils.core.components.Models;
 import com.wynntils.core.features.UserFeature;
+import com.wynntils.mc.event.ItemTooltipRenderEvent;
+import com.wynntils.wynn.handleditems.items.gui.ServerItem;
+import com.wynntils.wynn.objects.profiles.ServerProfile;
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Optional;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 public class LobbyUptimeFeature extends UserFeature {
     @Override
     public List<Model> getModelDependencies() {
-        return List.of(Models.ServerList, Models.ServerItemStack);
+        return List.of(Models.ServerList);
+    }
+
+    @SubscribeEvent
+    public void onTooltipPre(ItemTooltipRenderEvent.Pre event) {
```

Comment:
Are all item stack tooltips injected like this? I am sure this does not look great in allocations if we do this for gear items

Reply:
Tooltips are not handled in this PR. Having proper caching of tooltips is one reason why they are more tricky.